__tests__/groundplaces/createStopCluster.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { GroundPlaces } from '../../src/classes/groundplaces';
+
+describe('createStopCluster', () => {
+  it('should create and add a new StopCluster to the GroundPlacesList', () => {
+    const GroundPlacesInstance: GroundPlaces = new GroundPlaces({});
+
+    const infos = {
+      name: 'Paris, Île-de-France, France',
+      latitude: 49.00443,
+      longitude: 2.51703,
+      countryCode: 'fr',
+      type: 'cluster',
+    };
+
+    GroundPlacesInstance.createStopCluster(infos);
+
+    expect(GroundPlacesInstance.Storage.getGroundPlacesList()).toEqual({
```

Comment:
You're test is a bit too large.
If the method "getGroundPlacesList fails your teste will be failing. Though the creation might be good.
You should had other expects : 
* Expecte the GPUID function to be called 
* Mock the GPUID function ?
etc.

Reply:
This test was adapted